modules/space/rotation/spicerotation.cpp

```diff
@@ -96,6 +107,12 @@ SpiceRotation::SpiceRotation(const ghoul::Dictionary& dictionary)
         }
     }
 
+    if (p.fixedDate.has_value()) {
+        _fixedEphemerisTime = SpiceManager::ref().ephemerisTimeFromDate(*p.fixedDate);
+    }
+    _fixedDate = p.fixedDate.value_or("");
```

Comment:
We'd do `_fixedDate = p.fixedDate.value_or(_fixedDate)`  in order to not replicate the default value.   Basically:  If there is a value, use it, otherwise allocate `_fixedDate = _fixedDate` which preserves the default value set in the preamble of the constructor